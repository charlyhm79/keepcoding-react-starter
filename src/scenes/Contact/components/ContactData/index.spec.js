import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import ContactData from './';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<ContactData />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('Matchs snapshot', () => {
  const tree = renderer.create(<ContactData />).toJSON();
  expect(tree).toMatchSnapshot();
});
