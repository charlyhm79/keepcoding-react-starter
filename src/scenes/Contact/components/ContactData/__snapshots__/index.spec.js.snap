// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matchs snapshot 1`] = `
<div
  className="sc-bdVaJa XFGyz"
>
  <p
    className="sc-bwzfXH eNWKUP"
  >
    <strong>
      Nombre:
    </strong>
     Eric Weber
  </p>
  <p
    className="sc-bwzfXH eNWKUP"
  >
    <strong>
      Dirección:
    </strong>
     402 Price Lake, Kimberlyshire, WI 38712-1501
  </p>
  <p
    className="sc-bwzfXH eNWKUP"
  >
    <strong>
      Teléfono:
    </strong>
     (912) 526-5410 x4550
  </p>
  <p
    className="sc-bwzfXH eNWKUP"
  >
    <strong>
      Email:
    </strong>
     Eric_Weber@gmail.com
  </p>
</div>
`;
